src/components/Studio/Vacancies/Vacancy/Vacancy.module.css

```diff
@@ -0,0 +1,95 @@
+.vacancy {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  padding: 52px 32px 0;
+  background: var(--background-color);
+  z-index: 10;
+  overflow: auto;
+  -webkit-overflow-scrolling: touch;
+  pointer-events: all;
+
+  & :global(button) {
```

Comment:
Does this need a global, since it's just the name of an html element?

These buttons probably could use a `color: inherit` to be visible on after dark mode 
<img width="895" alt="image" src="https://user-images.githubusercontent.com/38729/195722843-77ed5e99-c64b-45f8-9b5e-fab5ab11601a.png">

Reply:
It actually doesn't, good spot. Have applied to all stylesheets process by postcss.

Also inheriting colours now.

Reply:
This also seems to conflict with `.applyButton` below where an `!important` is used to overwrite this